dbt/include/trino/macros/materializations/table.sql

```diff
@@ -1,6 +1,6 @@
 {% materialization table, adapter = 'trino' %}
   {%- set on_table_exists = config.get('on_table_exists', 'rename') -%}
-  {% if on_table_exists not in ['rename', 'drop'] %}
+  {% if on_table_exists not in ['rename', 'drop', 'replace'] %}
```

Comment:
do you plan to add it to snapshot as well as a part of this PR?

Reply:
I'll add it in separate PR